src/test/java/alinaTinca/tests/Tema5/Demoqa/Elements/UploadAndDownloadClass.java

```diff
@@ -0,0 +1,33 @@
+package alinaTinca.tests.Tema5.Demoqa.Elements;
+
+import AlinaTinca.BrowserManagerClass;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class UploadAndDownloadClass {
+
+    public static void main(String[] args) {
+        goToTheUploadAndDownload();
+        //check if the method called goToTheUploadAndDownload is done successfully
+    }
+
+    public static void goToTheUploadAndDownload(){
+        //create a method called goToTheUploadAndDownload that identifies a element and makes operation on it
+        ChromeDriver driver = BrowserManagerClass.createChromeDriver();
+        //create and configure the Chrome browser page settings
+        driver.get("https://demoqa.com/upload-download");
+        //go to this URL
+        WebElement UploadAndDownloadSection = driver.findElement(By.xpath("/html/body/div[2]/div/div/div[2]/div[1]/div/div/div[1]/div/ul/li[8]/span"));
+        //find the section called Upload and Download
+        UploadAndDownloadSection.click();
```

Comment:
This won't work with only a click. However, it's good that you tried. As a suggestion, this element has an input behind so you can use Selenium sendKeys() method to upload files.
![image](https://user-images.githubusercontent.com/18215854/211499829-9b3960f0-0b88-45c0-84a6-cd6ec9993868.png)